app.test.js

```diff
@@ -147,3 +147,40 @@ describe("/api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe(" /api/reviews/:review_id/comments", () => {
+  it("POST 201: should request body accepts an object with two properties: username and body. And should respond with posted comment.", () => {
+    const newComment = { username: "philippaclaire9", body: "cool game!" };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          review_id: 5,
+          author: "philippaclaire9",
+          body: "cool game!",
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it("GET 400: should respond with an error message indicating requested id is invalid. ", () => {
+    return request(app)
```

Comment:
you need to send something on this as well to prove that it's definitely testing the path